### ISSUE TITLE
Task bootstrap fails because of PROMO_GRAPHIC

### ISSUE THREAD

**Issue by NitroG42** [Triple-T/gradle-play-publisher#852]
### Describe the bug

When running the bootstrap task, it seems the Play API has changed because I get the following error:

```
      > A failure occurred while executing com.github.triplet.gradle.play.tasks.Bootstrap$ImageFetcher
         > 400 Bad Request
           GET https://www.googleapis.com/androidpublisher/v3/applications/myapp/edits/09106388581678554781/listings/fr-FR/promoGraphic
           {
             "code" : 400,
             "errors" : [ {
               "domain" : "global",
               "message" : "PROMO_GRAPHIC images are no longer supported.",
               "reason" : "badRequest"
             } ],
             "message" : "PROMO_GRAPHIC images are no longer supported.",
             "status" : "INVALID_ARGUMENT"
           }
```

### How To Reproduce
Just run bootstrap task
#### Versions

- Gradle Play Publisher: 2.8.0
- Gradle Wrapper: 6.5.0
- Android Gradle Plugin:  4.0.1

#### Tasks executed

./gradlew app:bootstrapRelease

### Expected behavior

No error in the task !

**Comment by NitroG42** [comment]
The task fails but the available resources are downloaded (which is a good news)

**Comment by sduduzog** [comment]
The thing is, after that, I couldn't publish at all

**Comment by SUPERCILEX** [comment]
Dang, they must have changed the API. If you can't publish right now, you'll need to delete the `promo-graphic` folder.

**Comment by sduduzog** [comment]
I hope it's not the fact that I've opted in to their new beta dashboard. I doubt the new flashy console UI didn't come with a whole API upgrade

**Comment by pavlospt** [comment]
Yeah API seems to have changed https://developers.google.com/android-publisher/api-ref/rest/v3/edits.images/list https://developers.google.com/android-publisher/api-ref/rest/v3/AppImageType

**Comment by Dohmanlechx** [comment]
We are getting failed CircleCI builds due to this issue as well.

**Comment by jpage4500** [comment]
@SUPERCILEX @Dohmanlechx - I'm also getting this issue running `./gradlew bootstrap`. I'm not trying to upload anything and don't have any `promo-graphic` folder locally. I just wanted to pull down the latest google play resources.

Also, I looked at the store presence and I don't have a promo graphic set here either.. thoughts?

![image](https://user-images.githubusercontent.com/1895553/92257906-f83c4200-eea3-11ea-9ca0-cede70081d68.png)

**Comment by jpage4500** [comment]
sorry, forgot to mention I'm running the latest version 2.8.0.. guessing the fix that was checked-in just isn't in that version yet

**Comment by SUPERCILEX** [comment]
Correct, the fix is in 3.0. If people really want it, I could create a patch release.

**Comment by jpage4500** [comment]
Sorry, when I first found this I noticed the issue was closed and assumed it was fixed.. my bad! I don't think it's going to prevent updating the app normally so I can wait until 3.0!

thanks

**Comment by ufuji1984** [comment]
@SUPERCILEX 

> If people really want it, I could create a patch release.

At least I (and my colleagues) want!!  🙇‍♂️  🙇‍♂️  🙇‍♂️

**Comment by SUPERCILEX** [comment]
Ok, will hopefully have time over the weekend.

**Comment by ufuji1984** [comment]
Very glad if you could !!! 😭  😭  😭

**Comment by SUPERCILEX** [comment]
Alrighty, I've just released 2.8.1 which should fix the issue.

**Comment by ufuji1984** [comment]
Thank you so much !!!
